src/components/Settings/MobileTab.react.js

```diff
@@ -145,6 +146,7 @@ class MobileTab extends React.Component {
               {countryData.countries.all[i].name +
                 ' ' +
                 countryData.countries.all[i].countryCallingCodes[0]}
+              <Emoji text={countryData.countries.all[i].emoji} />
             </MenuItem>
```

Comment:
how about ~~`country CallingCode flag`~~ ` flag CallingCode country`

Reply:
That's better. I'll fix it